certora/applyHarnessFifo.patch

```diff
@@ -1,146 +1,25 @@
 diff -ruN DoubleLinkedList.sol DoubleLinkedList.sol
---- DoubleLinkedList.sol	2023-05-23 16:01:50.888803463 +0200
-+++ DoubleLinkedList.sol	2023-05-23 16:44:01.777091764 +0200
-@@ -18,6 +18,8 @@
+--- DoubleLinkedList.sol	2024-08-14 10:10:08.974975205 +0200
++++ DoubleLinkedList.sol	2024-08-14 10:14:10.970436843 +0200
```

Comment:
Isn't there a way to remove dates in patch files? they don't bring anything and bring a lot of... diff

Reply:
There is a builtin way, it's the `--label` flag, but it does not apply to recursive diff. Fixed with df17d59 instead